extensions/wakatime/src/today-summary.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTodaySummary } from "./hooks";
+import { getDuration } from "./utils";
+
+import { MenuBarExtra, getPreferenceValues } from "@raycast/api";
+
+export default function Command() {
+  const todaySummary = useTodaySummary();
+  const preference = getPreferenceValues();
+  const showAsksSentence = preference.showAsksSentence;
+
+  function getTitle(): string | undefined {
+    const today = todaySummary.data;
+
+    if (today === undefined) return;
+
+    const cumulative_total = today.cummulative_total;
+    const todayText = getDuration(cumulative_total.seconds);
+
+    return cumulative_total.seconds === 0 && showAsksSentence ? undefined : `Today: ${todayText}`;
+  }
```

Comment:
Hey @Fatpandac @mattisssa
I've been travelling the past few days, so I saw the messages but couldn't do anything at the time.

I've checked it now and have found the reasons for the @mattisssa's errors.

I couldn't reproduce it, but I encountered a similar error as I'm working on #2434. WakaTime's documentation doesn't say this, but some fields can be `null`.

You can change your function to this.
```suggestion
  function getTitle(): string | undefined {
    const { seconds } = todaySummary?.data?.cummulative_total ?? {};
    if (seconds == undefined) return;
    
    return seconds === 0 && showAsksSentence ? undefined : `Today: ${getDuration(seconds)}`;
  }
```

I'll fix the same thing along with the `NaN%` in @Fatpandac's image, other issues and some new features in the PR above.

Reply:
Thx.